app/uk/gov/hmrc/apigatekeeperapprovalsfrontend/controllers/ConfirmResponsibleIndividualVerifiedController.scala

```diff
@@ -33,13 +33,30 @@ import uk.gov.hmrc.apigatekeeperapprovalsfrontend.domain.models.Standard
 import uk.gov.hmrc.apigatekeeperapprovalsfrontend.services.SubmissionReviewService
 import uk.gov.hmrc.apigatekeeperapprovalsfrontend.domain.models.SubmissionReview
 import org.joda.time.DateTime
+import play.api.data.Form
+import play.api.data.Forms._
 
 object ConfirmResponsibleIndividualVerifiedController {  
-  case class ViewModel1(appName: String, applicationId: ApplicationId, verified: Option[Boolean], errors: Option[String] = None) {
+  case class ViewModel(appName: String, applicationId: ApplicationId) {
   }
 
-  case class ViewModel2(appName: String, applicationId: ApplicationId, day: Option[String], month: Option[String], year: Option[String], errors: Option[String] = None) {
-  }
+  case class HasVerifiedForm(verified: String)
+
+  val hasVerifiedForm: Form[HasVerifiedForm] = Form(
+    mapping(
+      "verified" -> nonEmptyText
+    )(HasVerifiedForm.apply)(HasVerifiedForm.unapply)
+  )
+
+  case class VerifiedDateForm(day: String, month: String, year: String)
+
+  val verifiedDateForm: Form[VerifiedDateForm] = Form(
+    mapping(
+      "day" -> nonEmptyText,
```

Comment:
These should probably be number fields only

https://www.playframework.com/documentation/2.8.x/ScalaForms

Reply:
Fixed